include/logging/log_msg.h

```diff
@@ -74,7 +74,7 @@ struct log_msg_ids {
 	u16_t source_id : 10;   /*!< Source ID. */
 };
 
-_Static_assert(sizeof(struct log_msg_ids) == sizeof(u16_t),
+BUILD_ASSERT_MSG(sizeof(struct log_msg_ids) == sizeof(u16_t),
```

Comment:
I notice other uses of BUILD_ASSERT_MSG (in include/arch/arm/arch.h for example) the calls to this macro all have ``\`` at the end of the lines for continuation.  I suspect adding a ``\`` at the end of these continuations of the parameters to BUILD_ASSERT_MSG will remove the doxygen error.

Reply:
huh, good catch